### ISSUE TITLE
Change view - field from an edge - how to reference

### ISSUE THREAD

**Issue by moseleyi** [crudlio/crudl#64]
My GraphQL schema looks like this: 

```
User {
    username
    user_id
    user_detail {
        firstname
    }
}
```

When I define fields in changeView, user_id and username work but firstname is undefined. How do I reference it?
Also shouldn't the normalize function parameter (get) have all the data, just like listView?

**Comment by sehmaschine** [comment]
You can use `getValue`, see
https://github.com/crudlio/crudl-example-express/blob/master/crudl-admin-graphql/admin/views/entries.js#L257

And yes, normalize and denormalize should get all data/values.

**Comment by moseleyi** [comment]
Thank you I used getValue with select function (took some time to find it), but when I output the argument for normalize like this: 

```
normalize : (get) => {
    console.log(get);
}
```

it only outputs the fields I defined in the changeView fieldsets, but for listView that variable has all the data returned from the server. Is that expected behaviour?

**Comment by sehmaschine** [comment]
No, that's not expected behaviour. It should be consistent. We'll take a look.
